schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java

```diff
@@ -60,4 +66,290 @@ public static void main(String... args) {
     }
     System.exit(status);
   }
+
```

Comment:
Looks like there are a lot of code redundancy here. Can you please reduce it as much as possible?

Reply:
I refactored them in [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/542830245f1e97ab1b3be0f05e4f8e2fe11342a5)